packages/core/src/components/label/label.scss

```diff
@@ -0,0 +1,6 @@
+:host {
+  display: inline-block;
```

Comment:
Deze regel kun je volgens mij weghalen, bij default is het al `inline-block`

Reply:
Ik ga ervan uit dat het per default `display: inline;` is. Iig valt de styling om als je deze regel weghaalt.

Reply:
Ah, je hebt gelijk!